packaging/microshift/config.yaml

```diff
@@ -133,4 +133,5 @@ storage:
     # Allowed values are: unset, [], or one or more of ["snapshot-controller", "snapshot-webhook"]
     optionalCsiComponents:
         - ""
-
+# Disable default CNI so user can use different CNI than OVN-K
+DisableDefaultCNI: false
```

Comment:
Should we put it under the `network` section? 
Also, please change the name to `disableDefaultCNI` to be compatible with others.

Reply:
Yes, I think better to have it under network section because if it is disabled then any config on network section is not going to work. Also should we put the config name as `cniPlugin` and use same like storage `""` , "ovnk", and `"none" which will disable it?

Reply:
Yes, I like the consistency!

Reply:
so the default will remain,  if the value is unset it will defer to "ovnk" right?

Reply:
@eslutsky right, same as what we have for storage. I have updated this now.